libs/components/layout/src/lib/modules/back-to-top/back-to-top.directive.ts

```diff
@@ -29,8 +29,8 @@ export class SkyBackToTopDirective implements AfterViewInit, OnDestroy {
    * Specifies configuration options for the back to top component.
    */
   @Input()
-  public set skyBackToTop(value: SkyBackToTopOptions | undefined) {
-    this.#buttonHidden = !!value?.buttonHidden;
+  public set skyBackToTop(value: SkyBackToTopOptions | '' | undefined) {
+    this.#buttonHidden = !!(value as SkyBackToTopOptions)?.buttonHidden;
```

Comment:
Seems like this would fail if `value` were `undefined`. Could this be `!!(value && value?.buttonHidden)` to avoid the type coercion?

Reply:
We have a test that sets this input to `undefined`, so it definitely doesn't fail. But I like your implementation better because it's easier to read. Done!